### PR TITLE
Button functionality and Homepage Foundation

### PR DESCRIPTION
**Pull request by DSchriemer**
Hey guys,
I've created the foundation for our homepage as well as button functionality for opportunities. Please, if you have time, try and see if you can get the app running on your emulator. Feel free to call or text me if you're having issues!
(I added two screenshots to show what you should be seeing)
(ignore the lines and circles in screenshot two idk whats going on there)

Screen Shot 2020-03-31 at 10 44 14 PM
Screen Shot 2020-03-31 at 10 44 26 PM

### CODE REVIEW

File: app/src/main/java/com/example/volunteer/opportunities_activity.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.example.volunteer;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+
+public class opportunities_activity extends AppCompatActivity {
```
Comment (sdevalapurkar) [inline app/src/main/java/com/example/volunteer/opportunities_activity.java]: nit: naming convention for such files I believe would be better as:
`OpportunitiesActivity.java` and this class would be named `OpportunitiesActivity extends AppCompatActivity`

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -1,18 +1,66 @@
...
+        app:layout_constraintBottom_toTopOf="@+id/button2"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@mipmap/logo" />
 
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+    <Button
+        android:id="@+id/button5"
```
Comment (sdevalapurkar) [inline app/src/main/res/layout/activity_main.xml]: can we rename these buttons and give them ids that correspond to what they are?
ex:
`android:id="@+id/opportunitiesButton"`
`android:id="@+id/educationButton"`

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -1,18 +1,66 @@
...
+        android:id="@+id/button8"
+        style="@style/Widget.AppCompat.Button.Colored"
+        android:layout_width="348dp"
+        android:layout_height="92dp"
+        android:layout_alignParentBottom="true"
+        android:layout_centerInParent="true"
+        android:layout_marginBottom="77dp"
+        android:text="Tools" />
+
+</RelativeLayout>
```
Comment (sdevalapurkar) [inline app/src/main/res/layout/activity_main.xml]: nit: missing newline at end of file

File: app/src/main/res/layout/activity_opportunities_activity.xml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".opportunities_activity">
+
+
+    <Button
+        android:id="@+id/button2"
```
Comment (sdevalapurkar) [inline app/src/main/res/layout/activity_opportunities_activity.xml:11]: not sure what a better id for this button is but can we use something other than `button2`?

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (DSchriemer) [reply app/src/main/res/layout/activity_main.xml]: This pr is more just to see if my changes to the app are going to work on your side, once i can confirm this we can start using more proper naming conventions

Comment (sdevalapurkar) [reply app/src/main/res/layout/activity_main.xml]: sure, but if this is gonna go into master then we should update this before it does 😄

File: app/src/main/java/com/example/volunteer/opportunities_activity.java (hunk 1 above)
Comment (sdevalapurkar) [reply app/src/main/java/com/example/volunteer/opportunities_activity.java]: Screen Shot 2020-04-01 at 7 21 27 PM

**Review by sdevalapurkar — APPROVED** [review]
thanks for making the changes, LGTM

**Pull request merged**